packages/grafana-data/src/types/logs.ts

```diff
@@ -37,9 +37,19 @@ export interface LogsMetaItem {
 }
 
 export interface LogRowModel {
+  // Name of the filed from which the entry has been created so that we do not show it later in log row details.
+  entryField: string;
```

Comment:
Maybe save the index, not the name?  note that name does not have to be unique -- and once we start joining fields with the name 'value' it will be super common to have multiple fields with the same name (but different labels)

Reply:
hmm yes index probably makes more sense then. Is there some validation thought regarding the uniqueness or should there be? If not for name then name + field labels?

Reply:
I don't think we can or should assume names & labels are unique.  A DataFrame essentially maps to anything you could put in a CSV/spreadsheet.  Duplicate column names can happen -- and we should not explode.